### PR TITLE
Refactor setup

### PR DESCRIPTION
**Pull request by nuriel77**
Awaiting release of the action tools PR https://github.com/cloudbeds/composite-actions/pull/1
Once release and tagged, will modify the PR to reference the version (e.g. https://github.com/cloudbeds/composite-actions/compare/refactor-setup?expand=1#diff-b46666a384a80f8d9fc73776b1649c8881380fb92b3465ac365afa198fc78a7aR25)

### CODE REVIEW

File: helm-charts/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+      - uses: cloudbeds/composite-actions/helm-charts@v1
+        with:
+          charts: charts/nginx,charts/metallb,charts/app
+          document: true
+          push: true
+          kube_version: 1.19.3
+          username: "${{ secrets.CB_PROD_HELM_REPO_USERNAME }}"
+          password: "${{ secrets.CB_PROD_HELM_REPO_PASSWORD }}"
+          repo: https://charts.infra.cloudbeds-dev.com
```
Comment (svmt) [inline helm-charts/README.md:106]: repo is not required since it has default value

File: helm-charts/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  build:
+    name: Process Helm Chart
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - uses: cloudbeds/composite-actions/helm-charts@v1
+        with:
+          charts: charts/nginx,charts/metallb,charts/app
```
Comment (svmt) [inline helm-charts/README.md:100]: shouldn't this be resolved dynamically?

File: helm-charts/README.md.j2 [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+      - uses: cloudbeds/composite-actions/helm-charts@v1
+        with:
+          charts: charts/nginx,charts/metallb,charts/app
+          document: true
+          push: true
+          kube_version: 1.19.3
+          username: "{% raw %}${{ secrets.CB_PROD_HELM_REPO_USERNAME }}{% endraw %}"
+          password: "{% raw %}${{ secrets.CB_PROD_HELM_REPO_PASSWORD }}{% endraw %}"
+          repo: https://charts.infra.cloudbeds-dev.com
```
Comment (svmt) [inline helm-charts/README.md.j2]: same here wrong domain 
while it has correct default

File: helm-charts/README.md (hunk 2 above)
Comment (nuriel77) [reply helm-charts/README.md:100]: We moved the logic to the chart's dir, i.e.: https://github.com/cloudbeds/helmcharts-infra/blob/test-action-composite/.github/workflows/charts-ci.yaml#L30

File: helm-charts/README.md (hunk 1 above)
Comment (nuriel77) [reply helm-charts/README.md:106]: Good catch, will change it in the README.md.j2, that should reflect in the README.md

File: helm-charts/README.md.j2 (hunk 3 above)
Comment (nuriel77) [reply helm-charts/README.md.j2]: Updated in template, reflected in final [README.md](https://github.com/cloudbeds/composite-actions/blob/refactor-setup/helm-charts/README.md#full-usage-example) after auto doc

**Review by svmt — APPROVED** [review]

**Pull request merged**